engines/stark/ui/world/fmvscreen.cpp

```diff
@@ -53,7 +54,12 @@ FMVScreen::~FMVScreen() {
 	delete _surfaceRenderer;
 }
 
-void FMVScreen::play(const Common::String &name) {
+void FMVScreen::play(Common::String name) {
+	if (!StarkSettings->getBoolSetting(Settings::kHighFMV) && StarkSettings->hasLowResFMV()) {
+		name.erase(name.size() - 4);
+		name += "_lo_res.bbb";
+	}
```

Comment:
Is it possible that some versions of TLJ will use this suffix in upper-cases?

Reply:
It does not matter, the SearchManager used to lookup files in ResidualVM is completely case insensitive.